packages/account/src/utils/Utils.ts

```diff
@@ -55,7 +55,7 @@ export const compareChainIds = async (
   biconomySmartAccountConfig: BiconomySmartAccountV2Config,
   skipChainIdCalls: boolean,
 ): Promise<Error | void> => {
-  const signerResult = await convertSigner(signer, skipChainIdCalls);
+  const signerResult = await convertSigner(signer, skipChainIdCalls, biconomySmartAccountConfig.rpcUrl);
```

Comment:
why is this added?

Reply:
If a user passes a PrivateKeyAccount there is no info about transport so he needs to at least pass rpcUrl in the config so we can properly construct the walletClient with the correct transport